### PR TITLE
fix(timeseries-data-export): requests chunking and limit calculations

### PR DESCRIPTION
**Pull request by raikoss**
Use seriesNumber in datapoints fetching instead of only timeseriesIds.length

I'm opening a PR from the newest version in v1.x, since updating to v2.x needs SDK v3 which will break insight.
This change is identical to the one on v2.x, just without needing the new SDK version

I'm unsure which branch to merge to though, to get a version 1.21 out that we can use in Insight :)

### CODE REVIEW

File: src/components/TimeseriesDataExport/TimeseriesDataExport.tsx
Comment (valgussev) [inline src/components/TimeseriesDataExport/TimeseriesDataExport.tsx:138]: Is that because a series may contain multiple timeseries?

Comment (raikoss) [reply src/components/TimeseriesDataExport/TimeseriesDataExport.tsx:138]: SeriesNumber depends on both `timeseriesIds` and `timeseriesExternalIds`, and when I've been testing I've only passed `timeseriesExternalIds` for the charts, so limit became wrongly set

**Review by valgussev — APPROVED** [review]

**Review by polomani — COMMENTED** [review]
@mrtraser this is the code you have more context of?
I seems like we need v1 branch for backporting

**Comment by mrtraser** [comment]
Updated regarding chunking approach, which was wrong before. Can be merged and then released manually.

**Comment by raikoss** [comment]
Great, thanks for the help! I'm unsure what to do from here to push a new version, can someone guide me in the right direction? :smile:

**Comment by mrtraser** [comment]
This PR should be merged to the `v1` branch, but for now, it's blocked by CI, we already have a separate PR #676 to fix it. After merge with `v1`, I'll publish new version of the package (v1.20.1)

**Review by mrtraser — APPROVED** [review]
LGTM

**Comment by raikoss** [comment]
Then I'll merge it!

**Pull request merged**
